mmv1/templates/terraform/examples/parallelstore_instance_basic.tf.erb

```diff
@@ -0,0 +1,42 @@
+resource "google_parallelstore_instance" "<%= ctx[:primary_resource_id] %>" {
+  instance_id = "<%= ctx[:vars]['name'] %>"
+  location = "us-central1-a"
+  description = "test instance"
+  capacity_gib = 12000
+  network_config {
+    network_name = google_compute_network.network.name
+  }
```

Comment:
If `network_config` isn't supposed to be present it needs to be removed from here! This file is used to generate both documentation and acceptance tests.

Reply:
@SarahFrench Thanks modified the file!